include/seqan3/argument_parser/validators.hpp

```diff
@@ -352,6 +353,39 @@ class file_ext_validator // TODO AFTER INITIAL MERGE: Check if file exists, also
     std::vector<std::string> extensions;
 };
 
+/*!\brief A validator that checks if a file exists.
+ * \ingroup argument_parser
+ *
+ * \details
+ *
+ * The struct then acts as a functor that throws a seqan3::parser_invalid_argument
+ * exception whenever a given filename (string) does not exist.
+ *
+ * \snippet test/snippet/argument_parser/validators_3.cpp usage
```

Comment:
I think this is the snippet from the file_extension_validator. There needs to be a new one for this new validator :nerd_face: